src/content/reference/feature-flags.mdx

```diff
@@ -31,3 +31,4 @@ Below is a list of those variables that you can use to leverage these newer and
 | OKTETO_SUPPORT_STACKS_ENABLED                    | Support Okteto Stacks syntax                                                                                                                                                                                                    | `false`                |
 | OKTETO_SYNCTHING_VERSION                         | Specifies the Syncthing version the CLI must use                                                                                                                                                                                | N/A                    |
 | OKTETO_USE_STATIC_KUBETOKEN                      | Use a fixed token rather than a dynamic token for interactions with the Kubernetes API                                                                                                                                          | `false`                |
+| OKTETO_DEV_PERSISTENT_VOLUME_SIZE                | Override the default persistent volume size for dev containers                                                                                                                                                                  | N/A                    |
```

Comment:
Variables are organized alphabetically, could you move it to the right place, please?